frontend/src/__tests__/components/textinput.test.js

```diff
@@ -0,0 +1,188 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { SingleLineInput, MultiLineInput } from '../../../components';
+import Colors from '../../../components/colors';
+
+describe('Single and MultiLine Text Input Component Tests', () => {
+  describe('Single Line Text Input Component Tests', () => {
+    it('should render a default Single Line Input component', () => {
+      render(<SingleLineInput register={jest.fn()} errors={jest.fn()} />);
+
+      const textInputComponent = screen.getByRole('textbox');
+      expect(textInputComponent).toBeInTheDocument();
+      expect(screen.getByText('Label')).toBeInTheDocument();
+      expect(textInputComponent).toHaveStyle(
+        `color:${Colors.Mono[Colors.Mono.length - 1]}`,
+        'font-size: 1rem',
+        'height: 2rem',
+        'border-radius: 6px',
+        `border: 1px solid ${Colors.Mono[4]}`,
+        'padding-left: 0.5rem',
+        'box-sizing: border-box',
+        'width:100%'
+      );
+      expect(screen.getByTestId('div-1')).toHaveStyle('margin-bottom:20px');
+      expect(screen.getByText('Label')).toHaveStyle(
+        `color:${Colors.Mono[Colors.Mono.length - 1]}`
```

Comment:
Can you elaborate?

Reply:
This was originally meant to test the label which is above the input, however, i had changed the code so the label no longer has that style. I have adjusted the tests accordingly.